src/status_im/multiaccounts/key_storage/core.cljs

```diff
@@ -240,6 +241,23 @@ We don't need to take the exact steps, just set the required state and redirect
             (popover/hide-popover)
             (navigation/navigate-to-cofx :multiaccounts nil)))
 
+(fx/defn confirm-logout-and-goto-key-storage
+  {:events [::confirm-logout-and-goto-key-storage]}
+  [{:keys [db] :as cofx}]
+  (fx/merge cofx
+            {:db (assoc db :goto-key-storage? true)}
+            (multiaccounts.logout/logout)))
+
+(fx/defn logout-and-goto-key-storage
+  {:events [::logout-and-goto-key-storage]}
+  [_]
+  {:ui/show-confirmation
+   {:title               (i18n/label :t/logout-title)
+    :content             (i18n/label :t/logout-are-you-sure)
+    :confirm-button-text (i18n/label :t/logout)
+    :on-accept           #(re-frame/dispatch [::confirm-logout-and-goto-key-storage])
+    :on-cancel           nil}})
```

Comment:
`nil` is not callable, this might throw an error

Reply:
thanks! Seems to work fine, pressing the cancel button hides the pop-up and no error is displayed (even in logs)

Reply:
it's the same as you just do not add `:on-cancel` at all in the map